apis/v1/gateway_types.go

```diff
@@ -661,9 +671,9 @@ type TLSConfig struct {
 	//
 	// Support: Core
 	//
-	// +required
+	// +optional
 	// <gateway:experimental>
-	FrontendValidation FrontendTLSValidation `json:"frontendValidation"`
+	FrontendValidation *FrontendTLSValidation `json:"frontendValidation,omitempty"`
```

Comment:
If I'm following the nesting correctly, this will end up with `gateway.spec.tls.frontend.default.frontendValidation`. Let's simplify that to `gateway.spec.tls.frontend.default.validation`

On that note - it would be very helpful to have a new example in `/examples` that showed these new Gateway TLS fields fully filled out.

Reply:
yep, I will add the example once we agree on the form :)

Reply:
done